components/assetlist/index.css

```diff
@@ -157,33 +128,15 @@
 
 .spectrum-AssetList-itemChildIndicator {
 	display: none;
-	transition: transform ease var(--mod-assetlist-child-indicator-animation, var(--spectrum-assetlist-child-indicator-animation));
+	transition: transform ease var(--spectrum-animation-duration-100);
 	transform: var(--spectrum-logical-rotation,);
 }
 
 .spectrum-AssetList-itemLabel {
 	flex: 1;
-	padding-inline-start: var(--mod-assetlist-item-label-padding-inline-start, var(--spectrum-assetlist-item-label-padding-inline-start));
+	padding-inline-start: var(--spectrum-spacing-100);
 
 	text-overflow: ellipsis;
 	overflow: hidden;
 	white-space: nowrap;
 }
-
-@media (forced-colors: active) {
-	.spectrum-AssetList-item {
-		--highcontrast-assetlist-border-color-key-focus: Highlight;
-		--highcontrast-assetlist-item-background-color-hover: Highlight;
-		--highcontrast-assetlist-item-background-color-selected-hover: Highlight;
-		--highcontrast-assetlist-label-color: ButtonText;
-		--highcontrast-assetlist-item-background-color-selected: SelectedItem;
-
-		forced-color-adjust: none;
```

Comment:
I hope as we cultivate our relationship with design, we can also do more with accessibility, and try to have a better handle on our WHCM styles.